hardware_applications/STSL/include/STSL/LinuxUtils.h

```diff
@@ -0,0 +1,20 @@
+#ifndef TRAININGSUPPORTLIBRARY_LINUXUTILS_H
+#define TRAININGSUPPORTLIBRARY_LINUXUTILS_H
+
+#include "OSUtils.h"
+#include <vector>
+
+class LinuxUtils : public OSUtils {
+public:
+    virtual std::string FindRobot() override;
```

Comment:
is there a reason you're not using namespaces?

Reply:
You should never add a `using namespace ...` declaration to a header file, because it would include that namespace in every file which includes that header. This would cause a great deal of annoyance and confusion.

Reply:
I originally thought the first comment was referring to `using namespace *`, but I think it's referring to 'why can't these files be in their own namespace'?

Also `using namespace *` might in general be a bad idea (causes conflicts): https://github.com/RoboJackets/igvc-software/issues/140

Reply:
@sastorer Can you elaborate on the specifics of what you wanted here?
I'd thought from our offline conversation that you were looking for `using namespace std` here.